src/sentences/gst.rs

```diff
@@ -0,0 +1,137 @@
+use chrono::NaiveTime;
+
+use nom::{
+    character::complete::char,
+    number::complete::float,
+    combinator::opt,
+    IResult,
+};
+
+use crate::{parse::NmeaSentence, sentences::utils::parse_hms, Error, SentenceType};
+
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+// GST - GPS Pseudorange Noise Statistics
+//
+//              1    2 3 4 5 6 7 8   9
+//              |    | | | | | | |   |
+// $ --GST,hhmmss.ss,x,x,x,x,x,x,x*hh<CR><LF>
```

Comment:
```suggestion
/// ```text
///              1    2 3 4 5 6 7 8   9
///              |    | | | | | | |   |
/// $ --GST,hhmmss.ss,x,x,x,x,x,x,x*hh<CR><LF>
/// ```
```